spec/memo_wise_spec.rb

```diff
@@ -819,5 +819,28 @@ def test_method
         end
       end
     end
+
+    context "with module defined self.extended" do
+      let(:module_with_memo) do
+        Module.new do
+          prepend MemoWise
+
+          def self.extended(base)
+            base.instance_variable_set(:@extended_called, true)
+          end
+
+          def no_args; end
+          memo_wise :no_args
+        end
+      end
+
+      it "calls defined self.extended" do
+        klass = Class.new
+        instance = klass.new
+        instance.extend(module_with_memo)
+
+        expect(instance.instance_variable_get(:@extended_called)).to be(true)
```

Comment:
Hello @alpaca-tc thank you for this!

Can we add a test asserting, in addition to the fact that we call the previously defined `Module#extended`, that we also are actually memoizing the call to `no_args`? I suspect that was intended by maybe not finished?

Reply:
Added and force-pushed 👍